openslides/core/config_variables.py

```diff
@@ -57,6 +57,44 @@ def get_config_variables():
         group='General',
         subgroup='Event')
 
+    yield ConfigVariable(
```

Comment:
Why just not hardcode this into the `privacypolicy` page? I thing nobody should be allowed to change this.

Reply:
You should be able to change this, to maybe change some details for additional plugins, other languages and to add administrator mails or other contacts if you wan't to. But the best reason to change this, is if you host it by yourself and you delete logs after 10 days or something like this, this should then be changend in the privacy policy as well. As you can see the privacy policy must or should be changed according to every users setup and/or environment.

Reply:
Ok.